spaces/domains.json

```
@@ -792,5 +792,6 @@
   "vote.gurucollective.xyz": "gurucollective.eth",
   "dao.pinkieinu.com": "pinkieinu.eth",
   "vote.fluent.finance": "fluentdao.eth",
-  "fedvote.fluent.finance": "fluentfed.eth"
+  "fedvote.fluent.finance": "fluentfed.eth",
+  "vote.eledao.io":"ele-dao.eth"
```

Please add a space here.
```suggestion
  "vote.eledao.io": "ele-dao.eth"
```

---

ok，done